odoo/tests/common.py

```diff
@@ -950,6 +950,20 @@ def _reset(cb, funcs, data):
         self.cr.execute('SAVEPOINT test_%d' % self._savepoint_id)
         self.addCleanup(self.cr.execute, 'ROLLBACK TO SAVEPOINT test_%d' % self._savepoint_id)
 
+    @contextmanager
+    def enter_test_mode(self):
+        """
+        Enter test mode on the Registry where you can open test cursors on the
+        currently used registry by the test from ``self.env``.
```

Comment:
Doesn't sound too English to me: "on the currently used registry by the test from". I must confess I better understand the docstring I wrote for accounting.
> Make so that all new cursors opened on this database registry reuse the one currently used by the test.